example/react-native.config.js

```diff
@@ -1,10 +1,21 @@
 const path = require('path');
-const pak = require('../package.json');
+const pkg = require('../package.json');
 
 module.exports = {
+  project: {
+    ios: {
+      automaticPodsInstallation: true,
+    },
+  },
   dependencies: {
-    [pak.name]: {
+    [pkg.name]: {
       root: path.join(__dirname, '..'),
+      platforms: {
```

Comment:
This was a change I pulled from the new RN lib template, although it didn't seem to have any effect on our codegen issues.